### PR TITLE
Optimize Dockerfile

### PR DESCRIPTION
**Pull request by ilolm**
1. Reduced layers.
2. Enhanced readability.
3. Reduced image size.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,29 +1,28 @@
...
-    systemctl
-
-RUN wget https://as-repository.openvpn.net/as-repo-public.asc -qO /etc/apt/keyrings/as-repository.asc && \
-    echo "deb [arch=${TARGETPLATFORM#linux/} signed-by=/etc/apt/keyrings/as-repository.asc] http://as-repository.openvpn.net/as/debian jammy main">/etc/apt/sources.list.d/openvpn-as-repo.list && \
-    apt update && apt -y install openvpn-as=$VERSION && \
-    apt clean
-
-RUN mkdir -p /openvpn /ovpn/tmp /ovpn/sock && \
-    sed -i 's#~/tmp#/ovpn/tmp#g;s#~/sock#/ovpn/sock#g' /usr/local/openvpn_as/etc/as_templ.conf
+FROM ubuntu:22.04
```
Comment (uddr) [inline Dockerfile:4]: any reason `--platform` has been removed?

File: Dockerfile [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
+
+LABEL org.opencontainers.image.authors="pkg@openvpn.net" \
+      org.opencontainers.image.description="docker openvpn-as"
```
Comment (uddr) [inline Dockerfile:7]: putting in the description container name seems does not provide any value here

File: Dockerfile [hunk 3]
```diff
@@ -1,29 +1,28 @@
...
+
+ARG TARGETPLATFORM \
+    VERSION \
+    DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update \
```
Comment (uddr) [inline Dockerfile:13]: I understand that putting 3 `RUN` into one reduces layers but what's the value here?
There are so few of them.
I would say having 3 layers here have more benefits, for example save some time during rebuilding

File: Dockerfile [hunk 4]
```diff
@@ -1,29 +1,28 @@
...
+    && apt-get install -y ca-certificates wget gnupg net-tools iptables systemctl && \
+
+    echo "Adding openvpn repository." \
```
Comment (uddr) [inline Dockerfile:16]: if we have it as a separate layer we don't need this echo here

**Review by uddr — CHANGES_REQUESTED** [review]
@ilolm left some comments

File: Dockerfile (hunk 1 above)
Comment (ilolm) [reply Dockerfile:4]: its set by default

File: Dockerfile (hunk 3 above)
Comment (ilolm) [reply Dockerfile:13]: As you wish)

**Review by uddr — APPROVED** [review]

**Pull request merged**